apps/docs-snippets/src/guide/contracts/variable-outputs.test.ts

```diff
@@ -0,0 +1,35 @@
+import type { Contract } from 'fuels';
+import { getMintedAssetId, getRandomB256, Wallet } from 'fuels';
+
+import { DocSnippetProjectsEnum } from '../../../test/fixtures/forc-projects';
+import { createAndDeployContractFromProject } from '../../utils';
+
+/**
+ * @group node
+ */
+describe(__filename, () => {
+  let contract: Contract;
+  beforeAll(async () => {
+    contract = await createAndDeployContractFromProject(DocSnippetProjectsEnum.TOKEN);
+  });
+
+  it('should successfully execute contract call with txParams', async () => {
```

Comment:
```suggestion
  it('should successfully execute contract call with variable outputs', async () => {
```

Just making it more explicit as we have lots of tx param tests and #1961 introduces many